group.master

```diff
@@ -57,7 +57,7 @@ Debian-gdm:*:117:
 elasticsearch:*:118:
 ntp:*:119:
 metrics:*:120:
-systemd-journal:*:121:
+systemd-journal:*:121:gnome-initial-setup
```

Comment:
I think this may run into problems due to the weird interactions between `useradd` and `/lib/group`. New users are typically created in the `systemd-journal` group. What happens is that the group is in essence copied to `/etc/group` to make modifications. Once it exists at `/etc/group`, the entry in `/lib/group` is no longer used.

So, this would work until a new user is added.

Reply:
Although, it's possible that useradd is smart enough to maintain the current entries as it makes the new entry in `/etc/group`.

Reply:
Does this matter, in practice, in this case? This is for the benefit of the FBE user on live or reformatter USB sticks – if this group membership is lost when a user is added, that's fine, because if a human user is added we are definitely not in the FBE and about to run the reformatter.

Is there another approach?